tests/phantom/lib/actions/actionsSpec.js

```diff
@@ -8,6 +8,168 @@ describe('Using actions, the parser', () => {
         parser = createParser();
     });
 
+    it('condition.then', async () => {
```

Comment:
It's specs. Name of the suite + name of the test should form and a meaningful sentence.